### ISSUE TITLE
'AzureOpenAI' object is not callable

### ISSUE THREAD

**Issue by lingxiankong** [phidatahq/phidata#235]
Example code:

```
from phi.assistant import Assistant
from phi.llm.azure import AzureOpenAIChat

assistant = Assistant(
    llm=AzureOpenAIChat(model="gpt-35-turbo"),
    description="You help people with their health and fitness goals.",
)
assistant.print_response("Share a quick healthy breakfast recipe.", markdown=True)
```

Exception:

```
Traceback (most recent call last):
  File "/Users/kongli/code/testtest.py", line 11, in <module>
    assistant.print_response("Share a quick healthy breakfast recipe.", markdown=True)
  File "/Users/kongli/code/test.venv/lib/python3.11/site-packages/phi/assistant/assistant.py", line 1402, in print_response
    for resp in self.run(message=message, messages=messages, stream=True, **kwargs):
  File "/Users/kongli/code/test.venv/lib/python3.11/site-packages/phi/assistant/assistant.py", line 841, in _run
    for response_chunk in self.llm.response_stream(messages=llm_messages):
  File "/Users/kongli/code/test.venv/lib/python3.11/site-packages/phi/llm/openai/chat.py", line 615, in response_stream
    for response in self.invoke_stream(messages=messages):
  File "/Users/kongli/code/test.venv/lib/python3.11/site-packages/phi/llm/openai/chat.py", line 218, in invoke_stream
    yield from self.get_client().chat.completions.create(
               ^^^^^^^^^^^^^^^^^
TypeError: 'AzureOpenAI' object is not callable
```

Lib versions:

* phidata 2.4.2
* openai 1.27.0

**Comment by ashpreetbedi** [comment]
thanks @lingxiankong , checking

**Comment by lingxiankong** [comment]
@ashpreetbedi any update please?

**Comment by ashpreetbedi** [comment]
@ysolanky is testing this today

**Comment by ysolanky** [comment]
Hey @lingxiankong, we have identified the issue and it will be patched it the next release. Thank you for debugging this for us :)

**Comment by ashpreetbedi** [comment]
@lingxiankong this is now fixed in the v2.4.8 release. Thank you so much for the bug, we appreciate your patience and support so much :)
